lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3325,23 +3325,24 @@ void SwiftASTContext::CacheModule(swift::ModuleDecl *module) {
   m_swift_module_cache.insert({ID, module});
 }
 
-swift::ModuleDecl *SwiftASTContext::GetModule(const SourceModule &module,
-                                              Status &error) {
-  VALID_OR_RETURN(nullptr);
+std::pair<swift::ModuleDecl *, bool>
+SwiftASTContext::GetModule(const SourceModule &module, Status &error) {
```

Comment:
There is one call site that reads the `bool` half of the returned pair, all other callers want the `ModuleDecl`. One option is to add cached as an out-parameter (`bool *cached = nullptr`). Of course out parameters are kinda warty, but it suits this case ok.